apps/example/pages/_app.tsx

```diff
@@ -5,8 +5,4 @@ function CustomApp({ Component, pageProps }: AppProps) {
   return <Component {...pageProps} />;
 }
 
-export default withFronteggApp(CustomApp, {
-  /**
-   * Optional frontegg options
-   */
-});
+export default withFronteggApp(CustomApp, global.CYPRESS_CONFIG);
```

Comment:
just to make sure, when we do it this way, will we be able to also run the app without Cypress? just for debugging the package itself if needed and so on?